vpr/src/base/vpr_types.h

```diff
@@ -1736,6 +1736,7 @@ struct t_rr_node_route_inf {
     float acc_cost;
     float path_cost;
     float backward_path_cost;
+    float R_upstream;
```

Comment:
Can you please add a TODO here to investigate the affect of putting this here? Although it is better to put it in this struct, it is increasing the working set which can have some performance implications. This also may affect the serial performance which will make the Hybrid Connection Router less efficient (but that needs to be investigated).